src/api/todos.ts

```diff
@@ -0,0 +1,55 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+import { Filter } from '../types/Filter';
+
+export const USER_ID = 6657;
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (userId: number, todo: {}) => {
```

Comment:
add type for todo

Reply:
Not fixed. I guess it should has Todo type
```suggestion
export const postTodo = (userId: number, todo: Todo) => {
```

Reply:
still not fixed